calcom/cal.com
# fix: higher resolution Campsite icon

## What does this PR do?

In https://github.com/calcom/cal.com/pull/15713, I introduced the Campsite conferencing app. I incorrectly used a lower resolution version of our icon, so this PR fixes that by swapping it out with a higher resolution version of our icon.

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com). If N/A, write N/A here and check the checkbox. N/A

## How should this be tested?

Visit http://localhost:3000/apps/campsite to see the higher resolution version of the icon.
